docs/v0.6/versions/v0.6.md

```diff
@@ -0,0 +1,42 @@
+---
+id: v0.6
+title: riff v0.6.x
+sidebar_label: v0.6.x (snapshot)
+---
+
+<style>
+.mainContainer {
+  -webkit-animation: hideonload 2s;
+     -moz-animation: hideonload 2s;
+      -ms-animation: hideonload 2s;
+       -o-animation: hideonload 2s;
+          animation: hideonload 2s;
+}
+@keyframes hideonload {
+  from { opacity: 0; }
+  to   { opacity: 1; }
+}
+@-moz-keyframes hideonload {
+  from { opacity: 0; }
+  to   { opacity: 1; }
+}
+@-webkit-keyframes hideonload {
+  from { opacity: 0; }
+  to   { opacity: 1; }
+}
+@-ms-keyframes hideonload {
+  from { opacity: 0; }
+  to   { opacity: 1; }
+}
+@-o-keyframes hideonload {
+  from { opacity: 0; }
+  to   { opacity: 1; }
+}
+</style>
+
+[Get started with riff v0.6.x](../../v0.6/getting-started.md)
```

Comment:
can we use netlify redirects instead of these static hacks?

Reply:
maybe next time (it introduces lock-in)